newsletter.css

```diff
@@ -153,7 +154,7 @@
 
 .ef-webkit {
```

Comment:
nit: why are we creating our own things with webkit? It seems that it is an actual type of extension that some browsers support. This name is misleading.